examples/edit.ts

```diff
@@ -0,0 +1,252 @@
+import { createElement, ClassAttributes } from 'react';
+import * as ReactDOM from 'react-dom';
+const SparqlGenerator = require('sparqljs').Generator;
+const SparqlClient = require('sparql-http-client');
```

Comment:
Could we use `import` instead of `require`?

Reply:
But shouldn't we do that more consistently throughout the whole project? 
I'm all for it, but I want to limit the amount of time spent to improving the build and dependency issues

Reply:
I did some replacements ; I kept some `require` mostly when it was importing SVG or JSON files directly.
It should be good enough for now.